### DISCUSSION TITLE
Has anyone implemented GraphRAG with pdf as input?

### DISCUSSION THREAD

**Discussion by trishachander** [microsoft/graphrag#815] [Q&A]
The implementation tutorials I've come across implement with text, but perhaps someone has implemented it using pdfs as input using Unstructured's modules?

**Comment by l4b4r4b4b4** [comment]
yes. However Gotta be careful with messy pdfs as well as pdfs with tables, diagrams & images ;)

**Comment by 9prodhi** [comment]
I've come across [MinerU](https://github.com/opendatalab/MinerU) that might be beneficial for addressing your requirement. However, before proceeding, it's crucial to review its licensing terms to ensure compatibility with your project.
